src/components/menus/common-contextual-menu.tsx

```diff
@@ -41,7 +57,7 @@ export default function CommonContextualMenu({ menuItems, ...menuProps }: Readon
         disabled: boolean = false
     ) {
         return (
-            <MenuItem
+            <CustomMenuItem
```

Comment:
add 
                {icon && (
                    <ListItemIcon
                        style={{
                            minWidth: '25px',
                        }}
                    >
                        {icon}
                    </ListItemIcon>
                )}
in the below code